docs/Concepts/Plugins.md

```diff
@@ -0,0 +1,46 @@
+description: Plugins
+<!--- END of page meta data -->
+
+# Plugins
+
+You can extend Hyperledger Besu functionality by building Java plugins. The
+Plugin API allows you to take data from any Besu network, public or
+permissioned, and feed it into an application or system. 
+
+For example, a plugin can add additional monitoring functionality or stream
+event data to a third-party application. The API exposes data about the
```

Comment:
For example, a plugin can add additional  -> For example, create a plugin to add additional 

Suggested edit to make active

Reply:
Done.